server/container/mailbox-adapter/src/main/java/org/apache/james/adapter/mailbox/MailboxManagerManagement.java

```diff
@@ -142,6 +148,25 @@ public void deleteMailbox(String namespace, String user, String name) {
         }
     }
 
+    @Override
+    public void importEmlFileToMailBox(String namespace, String user, String name, String emlpath) {
+        checkMailboxArguments(namespace, user, name, emlpath);
+        MailboxSession session = null;
```

Comment:
You need to obtain a session from the mailboxManager: 

mailboxManager.createSystemSession will do the trick